data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -155,10 +155,10 @@
     field_names.ASTHMA_LOW_INCOME_FIELD: "ALI",
```

Comment:
I think all the `*_LOW_INCOME_FIELD`s also need to get updated to something like `*_LOW_INCOME_LOW_HIGHER_ED`.